firefox-ios/Client/Frontend/TrackingProtection/TrackingProtectionButton.swift

```diff
@@ -0,0 +1,108 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+import Common
+import ComponentLibrary
+
+public struct TrackingProtectionButtonViewModel {
+    let title: String
+    let a11yIdentifier: String
+
+    init(title: String = "",
+         a11yIdentifier: String = "") {
+        self.title = title
+        self.a11yIdentifier = a11yIdentifier
+    }
+}
+
+class TrackingProtectionButton: ResizableButton, ThemeApplicable {
+    private struct UX {
+        static let buttonCornerRadius: CGFloat = 12
+        static let buttonVerticalInset: CGFloat = 12
+        static let buttonHorizontalInset: CGFloat = 16
+        static let buttonFontSize: CGFloat = 16
+
+        static let contentInsets = NSDirectionalEdgeInsets(
+            top: buttonVerticalInset,
+            leading: buttonHorizontalInset,
+            bottom: buttonVerticalInset,
+            trailing: buttonHorizontalInset
+        )
+    }
+
+    private var viewModel = TrackingProtectionButtonViewModel()
+    private var backgroundColorNormal: UIColor = .yellow
+    private var foregroundColor: UIColor = .black
+    private var borderColor: UIColor = .black
```

Comment:
do we want to hardcode the values instead of using our theme system? What if we make this `.clear` so its obvious if there's an issue if we're not setting with our theme colors?